docs/smart-contracts/install/local-dev-node.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Local development network
+id: local-dev-node
+sidebar_position: 5
+description: "Start a local development network with a standalone blockchain node."
+---
+
+# Set up a development network
+
+Computers that run the Chainweb consensus protocol—the `chainweb-node` software—are a crucial component of the Kadena blockchain.
```

Comment:
Chainweb is not just consensus, it also provides the P2P network, mempool, etc. Maybe @chessai or Lars can comment on this here?